render.js

```diff
@@ -0,0 +1,27 @@
+const Renderer = () => {
+
+    const renderPosts = (posts) => {
+        $('#posts').text(' ')
+        // $('body').append('<div id ="posts"></div>')
+        
+        for (let post of posts) {
+            $('#posts').append(`<div class='post' id=${post.id} data-id=${post.id}></div>`)
+            $(`#${post.id}`).append(`<div class='post-text' id=${post.id} data-id=${post.id}>${post.text}</div>`)
+
+            for (let comment of post.comments) {
+                $(`#${post.id}`).append(`<div class='comments' data-id=${comment.id} id =${comment.id}></div>`)
```

Comment:
better practice is to create a variable that hold this jQuery element, and then append the two comments elements to this element, and in the end append this element